src/components/Events/TheEventsList.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+    <div class="events-list__container">
+        <div class='pagination-wrapper'>
+          <button
+            class='pagination__btn'
+            v-on:click='decrementCurrentPage'>decrement</button>
+
+          <h4>Week</h4>
+          <button
+            class='pagination__btn'
+            v-on:click='incrementCurrentPage'>increment</button>
+        </div>
+        <Event
+            v-for='event in pageOfEvents'
+            v-bind:key='event.id'
```

Comment:
You don't need the v-bind in front, its implied. So :key='incrementCurrentPage'.